schemas/stsci.edu/asdf/0.1.0/time/time.yaml

```diff
@@ -0,0 +1,232 @@
+
+            - `gps`: GPS time: seconds from 1980-01-06 00:00:00 UTC
+              For example, 630720013.0 is midnight on January 1,
+              2000.
+
+            - `unix`: Unix time: seconds from 1970-01-01 00:00:00
+              UTC. For example, 946684800.0 in Unix time is midnight
+              on January 1, 2000.  [TODO: Astropy's definition of
+              UNIX time doesn't match POSIX's here.  What should we
+              do for the purposes of ASDF?]
+
+            - `cxcsec`: Chandra X-ray Center seconds from 1998-01-01
+              00:00:00 TT. For example, 63072064.184 is midnight on
+              January 1, 2000.  [TODO: Is this general enough to
+              include?]
+
+          enum:
+            - iso
+            - yday
+            - byear
+            - jyear
+            - decimalyear
+            - jd
+            - mjd
+            - gps
+            - unix
+            - cxcsec
+
+        scale:
+          description: |
+            The time scale (or time standard) is a specification for
+            measuring time: either the rate at which time passes; or
+            points in time; or both. See also [3] and [4].
+
+            [TODO: Describing these scales in enough detail to
+            reimplement is perhaps too much to do here, but we should
+            have good ironclad references for each.]
+
+            The supported time scales are:
+
+            - `utc`: Coordinated Universal Time (UTC).  This is the
+              default time scale, except for the formats that
+              represents seconds since an epoch, namely `gps`, `unix`
+              and `cxcsec`.
+
+            - `tai`: International Atomic Time (TAI).
+
+            - `tcb`: Barycentric Coordinate Time (TCB).
+
+            - `tcg`: Geocentric Coordinate Time (TCG).
+
+            - `tdb`: Barycentric Dynamical Time (TDB).
+
+            - `tt`: Terrestrial Time (TT).
+
+            - `ut1`: Universal Time (UT1).
+
+          enum:
+            - utc
+            - tai
+            - tcb
+            - tcg
+            - tdb
+            - tt
+            - ut1
+
+        location:
+          description: |
+            Specifies the observer location for scales that are
+            sensitive to observer location, currently only `tdb`.  May
+            be specified either with geocentric coordinates (X, Y, Z)
+            or geodetic coordinates (longitude in degrees, latitude in
+            degrees, and optional height). [TODO: What are the units
+            for geocentric coordinates?]
+
+          anyOf:
+            - type: object
+              properties:
+                x:
+                  type: number
+                y:
+                  type: number
+                z:
+                  type: number
+              requiredProperties: [x, y, z]
+            - type: object
+              properties:
+                longitude:
+                  type: number
+                  minimum: -180
```

Comment:
In astropy Longitude can be specified by any input value and it is then taken modulo 360 to have 0 <= value < 360.  Do these `minimum` and `maximum` represent hard limits on allowed values?